ci-operator/step-registry/ipi/conf/vsphere/usertags/ipi-conf-vsphere-usertags-commands.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+echo "$(date -u --rfc-3339=seconds) - sourcing context from vsphere_context.sh..."
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/vsphere_context.sh"
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+if [ -f ${SHARED_DIR}/tags_lists ]; then
+  echo "$(date -u --rfc-3339=seconds) - additional tags defined, appending to platform spec"
+  PATCH="${SHARED_DIR}/additional_tags.yaml.patch"
+  cat >"${PATCH}" <<EOF
+platform:
+  vsphere:
+    failureDomains:
+    - topology:
+        tagIDs:
+$(cat ${SHARED_DIR}/tags_lists | awk -F "," '{print "          - "$2}')
+EOF
+
+yq-go m -x -i "${CONFIG}" "${PATCH}"
+else
+  echo "not found tags_lists, skip config tags"
+  cat ${SHARED_DIR}/tags_lists
```

Comment:
This else logic is strange, if "not found tags_lists", why we are trying to cat it?

Reply:
oh, thanks, that's for debug. to check whether these tags write into the file. will delete.